masonry/src/text/render_text.rs

```diff
@@ -60,6 +61,63 @@ pub fn render_text(
                         }
                     }),
                 );
+            // Draw the underline/strikethrough on top of the
```

Comment:
"on top of the glyphs"?

Reply:
I assume this means on top in depth order.

Slightly related: it would be nice to have descender skipping underlines but I think we’d need to do that in vello.